### PR TITLE
Add a general interface in md2ipynb for disable computing

### PR DESCRIPTION
**Pull request by StrayBird-ATSH**
## Description ##
As we want to simplify the process of compiling the docs and website manually on a laptop machine without GPU (e.g. MacBook Pro), we want a command that can disable computing python scripts in the docs. 

This pull request is a standalone change to the interface of md2ipynb file that supports converting without computing python scripts. 

This change is compatible with all the existing CI and Makefile. It accepts existing commands and enables computing on default. 

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

cc @dmlc/gluon-nlp-team

### CODE REVIEW

File: docs/md2ipynb.py [hunk 1]
```diff
@@ -4,7 +4,13 @@
 import notedown
 import nbformat
 
-assert len(sys.argv) == 2, 'usage: input.md'
+help_message = 'Usage:' \
```
Comment (leezu) [inline docs/md2ipynb.py]: You can use https://docs.python.org/3/library/argparse.html

**Comment by mli** [comment]
Job PR-1191/6 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1191/6/index.html

**Comment by mli** [comment]
Job PR-1191/8 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1191/8/index.html
